runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -469,7 +469,7 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
       if (sigLen < 1024)
          {
          char sigC[1024];
-         sigLen = sprintf(sigC, "%.*s.%.*s%.*s",
+         sigLen = snprintf(sigC, sizeof(sigC), "%.*s.%.*s%.*s",
```

Comment:
`sigLen` is not used after this assignment - it could be removed. Likewise at line 749.